packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -234,6 +264,9 @@ class InputDateTimePicker extends React.Component {
 								useTime={this.props.useTime}
 								useSeconds={this.props.useSeconds}
 								useUTC={this.props.useUTC}
+								onFinish={this.onFinish}
+								isValid={!this.state.errorMessage}
```

Comment:
It's more an error message than a valid flag.
And discussed together, it will be a list of errors definitions

Reply:
i'll remove this for now as we don't plan to disable the button and it was just for that for the moment.